### PR TITLE
Add custom 404 page

### PR DESCRIPTION
**Pull request by NullableInt**

### CODE REVIEW

File: 404.md [hunk 1]
```diff
@@ -2,8 +2,12 @@
 permalink: /404.html
 ---
 
-# Uh oh! Seems like you have taken a wrong turn somewhere
+<!-- markdownlint-disable MD026 -->
+# Uh oh:heavy_exclamation_mark: Seems like you have taken a wrong turn somewhere :construction:
+<!-- markdownlint-enable MD026 -->
 
-This page seems to try to hide from you.
+This page seems to try to hide from you. :dragon_face:
```
Comment (arebra) [inline 404.md:9]:
```suggestion
This page seems to play hide and seek with you. :dragon_face:
```
Just playing around :)

File: 404.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+---
+
+<!-- markdownlint-disable MD026 -->
+# Uh oh:heavy_exclamation_mark: Seems like you have taken a wrong turn somewhere :construction:
+<!-- markdownlint-enable MD026 -->
+
+This page seems to try to hide from you. :dragon_face:
+
+:circus_tent:
+Perhaps looking in the searchbar or clicking around in the menus will help?
```
Comment (asbjornu) [inline 404.md:12]:
```suggestion
Perhaps attempting a search or clicking around in the menus will help?
```

**Review by asbjornu — CHANGES_REQUESTED** [review]
Finally! 😍

**Review by arebra — APPROVED** [review]
👏 👏 👏

**Review by asbjornu — APPROVED** [review]

**Comment by asbjornu** [comment]
@NullableInt ❓

**Comment by NullableInt** [comment]
I used the "Finish feature" in SmartGit...

**Pull request closed without merging**